app/src/main/res/values/dimens.xml

```diff
@@ -51,7 +51,7 @@
     <dimen name="text_extra_small">12sp</dimen>
     <dimen name="text_small">16sp</dimen>
     <dimen name="text_normal">20sp</dimen>
-    <dimen name="text_large">28sp</dimen>
+    <dimen name="text_large">25sp</dimen>
```

Comment:
Did you check the places where this value is used before changing? Does this affect those ??